servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -2724,22 +2725,22 @@ void RenderForwardClustered::_update_render_base_uniform_set() {
 			RID sampler;
 			switch (decals_get_filter()) {
 				case RS::DECAL_FILTER_NEAREST: {
-					sampler = material_storage->sampler_rd_get_custom(RS::CANVAS_ITEM_TEXTURE_FILTER_NEAREST, RS::CANVAS_ITEM_TEXTURE_REPEAT_DISABLED);
+					sampler = material_storage->sampler_rd_get_default(RS::CANVAS_ITEM_TEXTURE_FILTER_NEAREST, RS::CANVAS_ITEM_TEXTURE_REPEAT_DISABLED);
```

Comment:
This is a change in behaviour that is worth discussing. I have a feeling that it makes sense for decals and projectors to have the per-viewport mipmap bias applied.

Reply:
I was unaware how these samplers are used to be honest to be sure. This should be a matter of just pulling from the other samplers below.

Reply:
You'll have to make the same change in the mobile renderer too.